frlearn/uncategorised/fuzzy_quantifiers

```diff
@@ -0,0 +1,413 @@
+from itertools import chain, combinations
+
+
+def choquet_integral(set_v, f, mu):
+    """"
+        Implementation of the Choquet_integral
+        parameters:
+        ----------
+        set_v: table (list)
+               The base set X on which the mathematics function f is defined
```

Comment:
Indeed, we can use two lists like f_val and mu_val as parmeters of this function, it will work correctly, but, for example, to evaluation a binary sentence using the Yager's model, we have to write many lines of code to get the differents values of the monotone measure. On the other hand, if we took the monotone measure to be a function and set_v to be a parameter, things would be clearer, I think. In fact, this is what motivated me to write the code as follows.